<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Leventure_DevelopKeyClass</class>
 <widget class="QMainWindow" name="Leventure_DevelopKeyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leventure_DevelopKey</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Designed by Leventure</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Leventure Devloper Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_welcom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 127);</string>
    </property>
    <property name="text">
     <string>大小姐驾到，统统闪开!</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>程序模式</string>
    </property>
    <widget class="QRadioButton" name="rabt_DeveloperMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>开发者模式</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rabt_InfoMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Info模式</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rabt_CompabilityModes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>兼容模式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rabt_TestMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>工程模式</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>调试工具</string>
    </property>
    <widget class="QComboBox" name="cobox_titlenames">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>窗口名称</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_fresh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_GetHwnd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>70</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>获得句柄</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_hwnd">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>70</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>窗口句柄:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_winmessage">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>发窗口消息:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sendmessage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txt_message">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_beginCode">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>消息号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>内容</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Github:LeventureQys</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Leventure_DevelopKey.qrc"/>
 </resources>
 <connections/>
</ui>
